### ISSUE TITLE
Firmware image type is broken

### ISSUE THREAD

**Issue by jlduran** [freebsd/poudriere#847]
## Prerequisites
- [x] Have you checked for an existing issue describing your problem?
- [x] Are you running the latest version?
- [x] Is your ports tree recent?
- [x] Is your FreeBSD Host on a [supported release](https://www.freebsd.org/security/#sup)?

## Describe the bug

The firmware image type is broken:

```
/usr/local/share/poudriere/image.sh: arithmetic expression: expecting ')': " ( 3725m - 32 - 32 - 0 ) / 2 "
```

I wonder if changing:

```diff
--- src/share/poudriere/image.sh
+++ src/share/poudriere/image.sh
@@ -719,7 +719,7 @@ usb)
 
 	# Figure out Partition sizes
 	OS_SIZE=
-	calculate_ospart_size "2" "${IMAGESIZE}" "${CFG_SIZE}" "${DATA_SIZE}" "${SWAPSIZE}"
+	calculate_ospart_size "2" "${NEW_IMAGESIZE_SIZE}" "${CFG_SIZE}" "${DATA_SIZE}" "${SWAPSIZE}"
 	# Prune off a bit to fit the extra partitions and loaders
 	OS_SIZE=$(( $OS_SIZE - 1 ))
 	WORLD_SIZE=$(du -ms ${WRKDIR}/world | awk '{print $1}')
```

as in 4d5dc5aa56c6283f994dd8e2d2226bb27aee812b together with something like 1f7c18bdc12fc48f80fd3f55c2250d766c035e6d and 79b43f316bc98b3d0e23d82ba6d8ced29dd7dd88 (but for `CFG_SIZE` and `DATA_SIZE`) is the way to go?

## How to reproduce

https://bsdrp.net/documentation/technical_docs/poudriere#the_6_minimum_steps_to_build_a_poudriere_firmware_image

## Expected behavior

A firmware image should be created.

**Comment by evadot** [comment]
I have some WIP code for poudriere image that fixes that.
I hope to push it this weekend.

**Comment by jlduran** [comment]
Closed by #874.
